compiler/optimizer/OMRSimplifier.cpp

```diff
@@ -337,6 +337,13 @@ OMR::Simplifier::simplifyExtendedBlock(TR::TreeTop * treeTop)
       if (block && !b->isExtensionOfPreviousBlock())
          break;
 
+      if (b->isOSRCodeBlock() || b->isOSRCatchBlock() || b->isOSRInduceBlock())
```

Comment:
Again, can we afford to omit tree simplification (one of the few opts we require for functional correctness) in a block that could have other trees before the OSR induction call ? I can understand this in the limited context of an OSR catch/code block but is it legal for a block ending in an OSR inductio call ?